### PR TITLE
Update lite.py

### PR DESCRIPTION
**Pull request by theangels**
If the model has it own loss function, the processing of the transforming will failed. So I set the compile=False when transform tf.keras model to tflite.

@wangtz

### CODE REVIEW

**Comment by gargn** [comment]
This PR changes the behavior of the converter for every user. If you want to use a model with this flag, please load your model into a Keras Session and use `TFLiteConverter.from_session` instead of using `TFLiteConverter.from_keras_model_file`.
